apis/Google.Cloud.Datastore.V1/Google.Cloud.Datastore.V1/PartitionIdPartial.cs

```diff
@@ -23,12 +23,14 @@ public partial class PartitionId
         /// </summary>
         /// <param name="projectId">The project ID of the partition. Must not be null.</param>
         /// <param name="namespaceId">The namespace ID of the partition. Must not be null.</param>
-        public PartitionId(string projectId, string namespaceId = "") : this()
+        /// <param name="databaseId">The database ID of the partition. Must not be null.</param>
+        public PartitionId(string projectId, string namespaceId = "", string databaseId = "") : this()
```

Comment:
This is a breaking change. (This is why the diff is failing.)

Just add an overload that accepts all three parameters, none of them defaulted. In the next major version we can (if we want) change to just this signature.

Reply:
rectified.